src/components/Jobs/jobs.util.js

```diff
@@ -271,6 +280,13 @@ export const enrichRunWithFunctionTag = (
       if (funcs) {
         const tagsList = uniq(map(funcs, 'metadata.tag'))
         set(jobRun, 'ui.functionTag', tagsList.join(', '))
+
+        jobRun = {
+          ...jobRun,
+          runOnSpot: capitalize(funcs[0].spec.preemption_mode ?? ''),
+          nodeSelectorChips: parseKeyValues(funcs[0].spec.node_selector || {}),
+          priority: generateFunctionPriorityLabel(funcs[0].spec.priority_class_name ?? '')
+        }
```

Comment:
use the same logic that we have for function tag. These fields better to set to the `ui` property, because it's not a part of the run object